### DISCUSSION TITLE
Do you have customized themes to share?

### DISCUSSION THREAD

**Discussion by fcoury** [extrawurst/gitui#1414] [General]
Would be nice to have a collection of user created themes for other uses. Does anyone have any and are willing to share?

Here's how the default theme look in my current iTerm config:

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/1371/198706120-7b8847c5-c90c-4c71-86f2-4d81705d6a7b.png">

**Comment by omad** [comment]
There's a good one at the bottom of #1360

**Comment by emirror-de** [comment]
I created a light theme that is, in my opinion, also user friendly in dark ambience.

https://github.com/emirror-de/noriko

![image](https://github.com/extrawurst/gitui/assets/30552361/7be6de31-cf37-4491-aaf3-290a86457c86)
